ckanext/ontario_theme/fanstatic/fonts_googleapis.css

```diff
@@ -0,0 +1,721 @@
+/* cyrillic-ext */
```

Comment:
Do we need to include the google fonts, instead of having the client pull them from the url (delete line of code in `ckanext/ontario_theme/templates/internal/base.html`)